src/04-arrays.js

```diff
@@ -24,7 +27,10 @@ function createEmptyArray() {}
  *  createArrayWithTwoElements(true, false);
  *  //> [ true, false ]
  */
-function createArrayWithTwoElements(a, b) {}
+function createArrayWithTwoElements(a, b) {
+  let arr = ["A", "B"];
```

Comment:
While this passes the tests  how can we write this so that whatever arguments are passed end up in the returned array?